libnavui-androidauto/src/main/java/com/mapbox/androidauto/deeplink/GeoDeeplinkPlacesListOnMapProvider.kt

```diff
@@ -42,9 +40,4 @@ class GeoDeeplinkPlacesListOnMapProvider(
     override fun cancel() {
         geoDeeplinkGeocoding.cancel()
     }
-
-    override fun searchOptions(): CarFeedbackSearchOptions = CarFeedbackSearchOptions(
-        geoDeeplink = geoDeeplink,
-        geocodingResponse = geocodingResponse,
```

Comment:
Looks like we don't set `geocodingResponse` anymore. There is no such function in `CarFeedbackRepository`.

Reply:
Oh right it's not passed from here https://github.com/mapbox/mapbox-navigation-android/blob/326af7da105ecd74e3cabb7517158de627b90674/libnavui-androidauto/src/main/java/com/mapbox/androidauto/deeplink/GeoDeeplinkPlacesListOnMapProvider.kt#L36

Reply:
Thanks for catching! It's added